### PR TITLE
HM 2841 dead link in opportunity builder

### PR DESCRIPTION
**Pull request by Revit**

### CODE REVIEW

File: apps/marketplace/pages/BriefResponsePage.js [hunk 1]
```diff
@@ -248,6 +248,7 @@ class BriefResponsePage extends Component {
                       this.props.changeModel(`${this.props.model}.${field}`, `${withGst}`)
                     }}
                     fileCount={this.state.otherDocumentFileCount}
+                    buttonAddedCount={this.state.otherDocumentFileCount}
```
Comment (tobiasbrown) [inline apps/marketplace/pages/BriefResponsePage.js:251]: This file shouldn't be changing as part of this work.  Can you please revert these changes?

File: apps/marketplace/components/Brief/BriefSpecialistResponseForm2.js [hunk 2]
```diff
@@ -367,7 +368,7 @@ const BriefSpecialistResponseForm2 = ({
                       addOtherDocument()
                     }}
                   >
-                    Add another
+                    Add another{buttonAddedCount}
```
Comment (tobiasbrown) [inline apps/marketplace/components/Brief/BriefSpecialistResponseForm2.js:371]: This file shouldn't be changing as part of this work.  Can you please revert these changes?

Comment (Revit) [reply apps/marketplace/components/Brief/BriefSpecialistResponseForm2.js]: reverted codes to master

File: apps/marketplace/pages/BriefResponsePage.js (hunk 1 above)
Comment (Revit) [reply apps/marketplace/pages/BriefResponsePage.js]: reverted to master codes

File: apps/marketplace/components/BuyerTraining/BuyerTrainingIntroductionStage.js [hunk 3]
```diff
@@ -22,7 +22,11 @@ const BuyerTrainingIntroductionStage = props => (
     <ul>
       <li>
         The{' '}
-        <a href="/api/2/r/learning-design-standards" rel="noopener noreferrer" target="_blank">
+        <a
```
Comment (tobiasbrown) [inline apps/marketplace/components/BuyerTraining/BuyerTrainingIntroductionStage.js:25]: This link can be updated in the database.  See if you can work out where it's updated.

**Review by tobiasbrown — APPROVED** [review]

**Comment by Revit** [comment]
Releasing task
